tests/acceptance/bootstrap/CollaborationContext.php

```diff
@@ -119,4 +117,47 @@ public function userCreatesFileInsideFolderInSpaceUsingWopiEndpoint(string $user
 			)
 		);
 	}
+
+	/**
+	 * @When user :arg1 tries to check the information of file :arg2 of space :arg3 using office :arg4 with invalid file-id
```

Comment:
```suggestion
	 * @When user :user tries to check the information of file :file of space :space using office :app with invalid file-id
```

Reply:
nice finding didn't know this doesn't fail tests 😸

Reply:
this thing is a valid code. so the test won't fail.